promptsource/templates/GEM/totto/templates.yaml

```diff
@@ -0,0 +1,273 @@
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: true
+    name: summarize_the_highlighted_cell
+    reference: ''
+  7a2fb34a-c792-41b3-9682-fcda15f1d100: !Template
+    answer_choices: null
+    id: 7a2fb34a-c792-41b3-9682-fcda15f1d100
+    jinja: '{% if table_section_title != "" %}
+
+      Given the following details:
+
+      Table Page Title: {{table_page_title}},
+
+      {% if table_section_text != "" %}
+
+      Table Section Text: {{table_section_text}},
+
+      {% endif %},
+
+      and the following original text describing the highlighted cell of the table:
+
+      {{sentence_annotations[0]["original_sentence"]}},
+
+      Can you guess the table''s section title?
+
+      |||
+
+      {{table_section_title}}
+
+      {% endif %}
+
+      '
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: false
+    name: guess the table section title
+    reference: ''
+  97930f85-12ae-4636-96ce-95edd05b66bf: !Template
+    answer_choices: null
+    id: 97930f85-12ae-4636-96ce-95edd05b66bf
+    jinja: '{% if target != "" %}
+
+      A table named "{{table_page_title}}" with section "{{table_section_title}}"
+      has the following highlighted cells:
+
+      {% for cell in highlighted_cells: %}
+
+      {% if table[0]|length >= cell[1] and table[0]|length >= cell[0] %}
+
+      {{table[0][cell[1]]["value"]}}: {{table[cell[0]][cell[1]]["value"]}}{% endif
+      %}{% endfor %}
+
+      Using the below text as reference, can you generate a summary depicting the
+      above information?
+
+
+      Text: {{sentence_annotations[0]["original_sentence"]}}
+
+      |||
+
+      {{target}}
+
+      {% endif %}
+
+      '
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: true
+    name: given_highlighted_cells_give_target_sentence
+    reference: ''
+  a78e0a87-c722-463f-bef6-c231108f8fdf: !Template
+    answer_choices: null
+    id: a78e0a87-c722-463f-bef6-c231108f8fdf
+    jinja: '{% if table_section_text != "" %}
+
+      Given the following details:
+
+      Table Page Title: {{table_page_title}},
+
+      Table Section Title: {{table_section_title}},
+
+      and the following original text describing the highlighted cell of the table:
+
+      {{sentence_annotations[0]["original_sentence"]}},
+
+      Can you guess the table''s section text?
+
+      |||
+
+      {{table_section_text}}
+
+      {% endif %}
+
+      '
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: false
+    name: guess the table section text
+    reference: ''
+  b25d135f-7c3a-4123-af40-2083a28a7add: !Template
+    answer_choices: null
+    id: b25d135f-7c3a-4123-af40-2083a28a7add
+    jinja: 'Convert the given linearized representation of a table to a coherent sentence:
```

Comment:
This language is a bit technical- @awebson has suggested that we should avoid language that is too technical to describe a prompt.  Could we perhaps say "Describe the given table in one sentence"?